.coveragerc

```diff
@@ -0,0 +1,14 @@
+[run]
+branch = True
+source = snorkel
+
+[report]
+exclude_lines =
+    pragma: no cover
+    raise NotImplementedError
+    if __name__ == .__main__.:
+    def __repr__
+ignore_errors = True
```

Comment:
?

Reply:
This is the generic .coveragerc suggested by codecov: https://gist.github.com/codecov-io/bf15bde2c7db1a011b6e